### PR TITLE
Aatams sattag dm update

### PR DESCRIPTION
**Pull request by xhoenner**
Related to https://github.com/aodn/data-services/pull/224.

Data bags need to be updated, new UUID for metadata context group is: 06b09398-d3d0-47dc-a54a-a745319fbece

### CODE REVIEW

**Comment by danfruehauf** [comment]
What UUID has to change?

**Comment by xhoenner** [comment]
The one for the Metadata context group.

File: workspace/AATAMS_SATTAG_DM/context/Source_0.1.item [hunk 1]
```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xmi:XMI xmi:version="2.0" xmlns:xmi="http://www.omg.org/XMI" xmlns:talendfile="platform:/resource/org.talend.model/model/TalendFile.xsd">
   <talendfile:ContextType confirmationNeeded="false" name="Default">
-    <contextParameter comment="" name="unarchived_folder" prompt="unarchived_folder?" promptNeeded="false" type="id_String" value="/usr/local/talend/aatams_sattag_dm/unzipped"/>
+    <contextParameter comment="" name="unarchived_folder" prompt="unarchived_folder?" promptNeeded="false" type="id_String" value="/Users/xavierhoenner/Work/chef/tmp/aatams_sattag_dm/unzipped"/>
```
Comment (danfruehauf) [inline workspace/AATAMS_SATTAG_DM/context/Source_0.1.item]: Umn. Maybe don't change the default here? At least it doesn't have to be committed.

File: workspace/AATAMS_SATTAG_DM/process/aatams_sattag_dm_subjobs/Harvest_CTDdata_0.1.item [hunk 2]
```diff
@@ -14,7 +14,7 @@
     <contextParameter comment="" name="aatams_sattag_nrt_Schema" prompt="aatams_sattag_nrt_Schema?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value="aatams_sattag_nrt"/>
     <contextParameter comment="" name="exclude" prompt="exclude?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value=".*ct91.mdb|.*ct101.mdb|.*ct110.mdb"/>
     <contextParameter comment="" name="include" prompt="include?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value=".*.mdb"/>
-    <contextParameter comment="" name="unarchived_folder" prompt="unarchived_folder?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value="/usr/local/talend/aatams_sattag_dm/unzipped"/>
+    <contextParameter comment="" name="unarchived_folder" prompt="unarchived_folder?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value="/Users/xavierhoenner/Work/chef/tmp/aatams_sattag_dm/unzipped"/>
```
Comment (danfruehauf) [inline workspace/AATAMS_SATTAG_DM/process/aatams_sattag_dm_subjobs/Harvest_CTDdata_0.1.item]: Same here.

File: workspace/AATAMS_SATTAG_DM/process/aatams_sattag_dm_subjobs/Harvest_LocDiveSummaryHauloutData_0.1.item [hunk 3]
```diff
@@ -14,7 +14,7 @@
     <contextParameter comment="" name="aatams_sattag_nrt_Schema" prompt="aatams_sattag_nrt_Schema?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value="aatams_sattag_nrt"/>
     <contextParameter comment="" name="exclude" prompt="exclude?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value=".*ct91.mdb|.*ct101.mdb|.*ct110.mdb"/>
     <contextParameter comment="" name="include" prompt="include?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value=".*.mdb"/>
-    <contextParameter comment="" name="unarchived_folder" prompt="unarchived_folder?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value="/usr/local/talend/aatams_sattag_dm/unzipped"/>
+    <contextParameter comment="" name="unarchived_folder" prompt="unarchived_folder?" promptNeeded="false" repositoryContextId="_DepiIAh_EeO-m4WFJQNDNA" type="id_String" value="/Users/xavierhoenner/Work/chef/tmp/aatams_sattag_dm/unzipped"/>
```
Comment (danfruehauf) [inline workspace/AATAMS_SATTAG_DM/process/aatams_sattag_dm_subjobs/Harvest_LocDiveSummaryHauloutData_0.1.item]: Yep, just everywhere.

**Comment by danfruehauf** [comment]
@xhoenner Fix those little defaults and I'll merge it.

**Comment by xhoenner** [comment]
see https://github.com/aodn/harvesters/commit/8720d55986a8f2d6174406ba143a990cfeeeccae @danfruehauf

**Comment by xhoenner** [comment]
The harvester works on 14-nsp, could you deploy it to 10-nsp @danfruehauf ?

**Comment by danfruehauf** [comment]
Was already deployed.

We need:
https://github.com/aodn/data-services/pull/239

**Pull request merged**